webcompat/static/js/lib/issues.js

```diff
@@ -547,8 +547,10 @@ issues.MainView = Backbone.View.extend(
       var target =
         e.target.nodeName === "IMG"
           ? e.target
-          : e.target.nodeName === "P" && e.target.firstElementChild;
+          : e.target.nodeName === "P" && e.target.querySelectorAll("img");
```

Comment:
@miketaylr is that what you expected?

Reply:
@magsout almost! we should use [`querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector), not [`querySelectorAll`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll). The former returns an Element, and the latter returns a NodeList.

Reply:
okie, sry for that. I think it's fixed now.
I did something wrong the first time I tested with querySelector and it wanst working that's why I tried with querySelectorAll.